modules/Bio/EnsEMBL/Slice.pm

```diff
@@ -3759,6 +3759,27 @@ sub add_synonym{
   return;
 }
 
+=head2 feature_so_acc
+
+  Example     : $slice_so_acc = $slice->feature_so_acc;
+  Description : This method returns a string containing the SO accession number of the slice, based on the coordinate system name.
+  Returns     : string (Sequence Ontology accession number)
+=cut
+
+sub feature_so_acc {
+  my $self = shift;
+
+  my %region_so_mapping = (
```

Comment:
As above, take this outside of the sub definition, so that it is not defined for every invocation.